src/mono/mono/mini/cpu-arm64.mdesc

```diff
@@ -504,6 +504,7 @@ xcompare_fp: dest:x src1:x src2:x len:4
 negate: dest:x src1:x len:4
 ones_complement: dest:x src1:x len:4
 xbinop_forceint: dest:x src1:x src2:x len:4
+xcast: dest:x len:4
 
```

Comment:
This needs an src1:x and a clob:1 so it doesn't need to emit a move.

Reply:
fixed